### PR TITLE
AST-1556 - Added remove clone items trigger for extending sortable item & fixed QA comments

### PR DESCRIPTION
**Pull request by imnavanath**
### Description
- Added remove clone items trigger for extending sortable item & fixed QA comments
- Ref gist prepared for future help - https://gist.github.com/imnavanath/1e387d2c8cf33c906a9d6160c7e1d54b

### Screenshots
- https://share.bsf.io/geud9ZoA

### Types of changes
- New feature (non-breaking change)

### How has this been tested?
- New extending sortable control for nested sub-controls

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: inc/customizer/extend-custom-controls/src/list-icons/item-component.js [hunk 1]
```diff
@@ -35,32 +35,40 @@ const ItemComponent = props => {
...
+						featuredItems.find( '.ahfb-sorter-item-remove.hide' ).toggleClass( 'hide' );
 					}}>
 				<Dashicon icon="admin-page"/>
 			</Button>
-			<Button className={`ahfb-sorter-item-expand ${props.item.enabled ? 'item-is-visible' : 'item-is-hidden'}`}
+			<Button className={`ahfb-sorter-item-remove ${props.item.enabled ? 'item-is-visible' : 'item-is-hidden'}`} isDestructive onClick={e => {
+				e.stopPropagation();
+				props.removeItem(props.index);
+
+				let featuredItems = jQuery( e.target ).closest( '.ahfb-sorter-drop-social_item_group' ).find( '.ahfb-sorter-item' );
```
Comment (premanshup) [inline inc/customizer/extend-custom-controls/src/list-icons/item-component.js:58]:
```suggestion
				const featuredItems = jQuery( e.target ).closest( '.ahfb-sorter-drop-social_item_group' ).find( '.ahfb-sorter-item' );
```

File: inc/customizer/extend-custom-controls/src/sortable/control.js [hunk 2]
```diff
@@ -139,14 +147,37 @@ export const sortableControl = wp.customize.astraControl.extend( {
...
+			mainIndexClonner.show();
+		} else {
+			control.sortableContainer.find( '.ast-sortable-item[data-value="' + originalItem + '"]' ).find( '.dashicons-visibility' ).after( '<i class="dashicons sortable-clonner dashicons-admin-page"></i>' );
+		}
+
+		var sortableItems = control.sortableContainer.find( '.ast-sortable-item[data-index="' + originalItem + '"]' );
+		if( sortableItems.length && 1 < sortableItems.length ) {
+			parent_wrap.remove();
+		}
+
```
Comment (premanshup) [inline inc/customizer/extend-custom-controls/src/sortable/control.js:167]: @imnavanath Please convert this CSS to ES6.

File: inc/customizer/extend-custom-controls/src/sortable/control.js [hunk 3]
```diff
@@ -139,14 +147,37 @@ export const sortableControl = wp.customize.astraControl.extend( {
...
+		control.addListeners( control );
+
```
Comment (premanshup) [inline inc/customizer/extend-custom-controls/src/sortable/control.js:169]:
```suggestion
```

File: inc/customizer/extend-custom-controls/src/sortable/control.js [hunk 4]
```diff
@@ -139,14 +147,37 @@ export const sortableControl = wp.customize.astraControl.extend( {
 		control.updateValue();
 	},
 
+	removeSortableItem: function( instance, control ) {
+
+		var parent_wrap = jQuery( instance ).closest( '.ast-sortable-item' ),
```
Comment (premanshup) [inline inc/customizer/extend-custom-controls/src/sortable/control.js:152]:
```suggestion
		var parentWrap = jQuery( instance ).closest( '.ast-sortable-item' ),
```

Comment (premanshup) [inline inc/customizer/extend-custom-controls/src/sortable/control.js:152]:
```suggestion
		var parentWrap = jQuery( instance ).closest( '.ast-sortable-item' ),
```

File: inc/customizer/extend-custom-controls/src/sortable/control.js [hunk 5]
```diff
@@ -139,14 +147,37 @@ export const sortableControl = wp.customize.astraControl.extend( {
...
 	addListeners: function( control ) {
-		let cloneTriggers = document.getElementsByClassName( 'clonned-sortable-item' ),
+		let removeTriggers = document.getElementsByClassName( 'remove-sortable-item' ),
 			accordionTriggers = document.getElementsByClassName( 'clonned-sortable-accordion' ),
 			visibilityTriggers = document.getElementsByClassName( 'clonned-sortable-visibility' );
 
-		for( var i=0; i < cloneTriggers.length; i++ ) {
-			cloneTriggers[i].onclick = function() {
-				control.cloneSortableItem( this, control );
+		for( var i=0; i < removeTriggers.length; i++ ) {
```
Comment (premanshup) [inline inc/customizer/extend-custom-controls/src/sortable/control.js:178]:
```suggestion
		for( var i=0; i < removeTriggers.length; i++ ) {
```
@imnavanath Please use proper variable names.

Comment (imnavanath) [reply inc/customizer/extend-custom-controls/src/sortable/control.js:174]: Done

File: inc/customizer/extend-custom-controls/src/sortable/control.js (hunk 4 above)
Comment (imnavanath) [reply inc/customizer/extend-custom-controls/src/sortable/control.js]: Done

Comment (imnavanath) [reply inc/customizer/extend-custom-controls/src/sortable/control.js]: Done

File: inc/customizer/extend-custom-controls/src/sortable/control.js (hunk 2 above)
Comment (imnavanath) [reply inc/customizer/extend-custom-controls/src/sortable/control.js]: Done

**Review by premanshup — APPROVED** [review]

**Pull request merged**